rand_distr/src/binomial.rs

```diff
@@ -13,6 +13,8 @@ use crate::{Distribution, Uniform};
 use rand::Rng;
 use core::fmt;
 use core::cmp::Ordering;
+#[allow(unused_imports)]
+use num_traits::Float;
```

Comment:
It's a bit ugly, but we currently don't have a way of checking whether `std` enabled for current build, note that it's not equivalent to checking `feature = "std"`.